File: FertiWeb/src/main/java/dto/DTOTipoFuente.java
package dto;

import dominio.Elemento;
import dominio.Unidad;
import lombok.Data;

@Data
public class DTOTipoFuente {

	private Long codigo;
	private String nombre;
	private Double aporte;
	private Unidad unidad;
	private Boolean estado;
	private Elemento elemento;
}

File: FertiMobile/src/app/dto/dto-tipo-fuente.ts
import { Unidad } from './../dominio/unidad';
import { Elemento } from '../dominio/elemento';
export class DTOTipoFuente {
    codigo: number;
    nombre: string;
    aporte: number;
    unidad: Unidad;
    estado: boolean;
    elemento: Elemento;
}
